srcpkgs/flatpak/template

```diff
@@ -31,10 +32,18 @@ build_options_default="gir"
 
 system_accounts="_flatpak"
 
+if [ -z "$XBPS_CHECK_PKGS" ]; then
+	configure_args+=" -Dtests=false"
+fi
+
 post_patch() {
 	vsed -i -e '/cp.*usr.lib.locale.C/d' tests/make-test-runtime.sh
 }
 
+do_check() {
+	meson test -C build --timeout-multiplier 3
+}
```

Comment:
could you use the default do_check from the build style?

Reply:
@paper42 how is it now? or i can patch the meson.build to increase timeout.